src/components/Editor/ConditionalPanel.js

```diff
@@ -60,10 +62,15 @@ export class ConditionalPanel extends PureComponent<Props> {
   };
 
   setBreakpoint(condition: string) {
-    const { selectedLocation, line } = this.props;
+    const { selectedLocation, location } = this.props;
```

Comment:
something like this can let us know for sure we have a Location and simplify the types

```diff
diff --git a/src/components/Editor/index.js b/src/components/Editor/index.js
index 45be88063..d4bb1d631 100644
--- a/src/components/Editor/index.js
+++ b/src/components/Editor/index.js
@@ -553,7 +553,7 @@ class Editor extends PureComponent<Props, State> {
         <HighlightLines editor={editor} />
         <EditorMenu editor={editor} />
         <GutterMenu editor={editor} />
-        <ConditionalPanel editor={editor} />
+        {conditionalPanelLocation ? <ConditionalPanel editor={editor} /> : null}
```